modules/editable-layers/src/utils/geojson-types.ts

```diff
@@ -11,64 +11,27 @@ import type {
   MultiPoint,
   MultiLineString,
   MultiPolygon,
-  Position
+  Position,
 } from 'geojson';
 
-export {Point, LineString, Polygon, MultiPoint, MultiLineString, MultiPolygon, Position};
+export { Point, LineString, Polygon, MultiPoint, MultiLineString, MultiPolygon, Position };
 
-export type PointCoordinates = Position;
-export type LineStringCoordinates = Position[];
-export type PolygonCoordinates = Position[][];
-export type MultiPointCoordinates = Position[];
-export type MultiLineStringCoordinates = Position[][];
-export type MultiPolygonCoordinates = Position[][][];
+export type SingleGeometry = Point | LineString | Polygon | MultiPoint | MultiLineString | MultiPolygon;
 
-export type AnyCoordinates =
-  | PointCoordinates
-  | LineStringCoordinates
-  | PolygonCoordinates
-  | MultiPointCoordinates
-  | MultiLineStringCoordinates
-  | MultiPolygonCoordinates;
-
-export type Geometry = Point | LineString | Polygon | MultiPoint | MultiLineString | MultiPolygon;
+export type AnyCoordinates = SingleGeometry['coordinates']
 
 export type Polygonal = Polygon | MultiPolygon;
 
 // Feature types
 
-export type BoundingBoxArray = [number, number, number, number];
+import { Feature, FeatureCollection } from 'geojson';
+export { Feature, FeatureCollection } from 'geojson';
```

Comment:
Probably want to drop this export, and have all files import directly from `geojson` instead - this was a quick way to make the switch.